packages/compiler/test/server/signature-help.test.ts

```diff
@@ -130,8 +162,24 @@ describe("compiler: server: signature help", () => {
     const wholeFile = `
       import "./dec-types.js";
       
+      /** 
```

Comment:
Is the colorization of docstring a separate problem?

Reply:
Yeah, we don't handle it. I will file an issue.

Reply:
#1347